docs/docs/api/Dispatcher.md

```diff
@@ -952,6 +952,38 @@ const client = new Client("http://example.com").compose(
 );
 ```
 
+##### `dump`
```

Comment:
Why did you call it `dump`? I would call it `head`, as a reference to the UNIX head command. `curl http://nodejs.org | head`.

Reply:
`dump` not a very "sexy" name but is related to the linux term [core dump](https://en.wikipedia.org/wiki/Core_dump). The proposed "head" is not self-explaining, especially it is about snapshotting the body of the response and not the head of the reponse.